### PR TITLE
Renaming in Azure Spring Monitor

### PR DESCRIPTION
**Pull request by jeanbisutti**
One motivation is to have more specific Spring bean names

### CODE REVIEW

File: eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml [hunk 1]
```diff
@@ -110,7 +110,7 @@
 
   <!-- AzureTelemetryConfig defines Spring bean used in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java-->
```
Comment (trask) [inline eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml:111]:
```suggestion
  <!-- AzureSpringMonitorConfig defines Spring bean used in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java-->
```

File: eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml [hunk 2]
```diff
@@ -174,6 +174,9 @@ the main ServiceBusClientBuilder. -->
   <suppress checks="com.azure.tools.checkstyle.checks.ThrowFromClientLoggerCheck"
             files=".*[/\\]com[/\\]azure[/\\]core[/\\]util[/\\]logging[/\\]*"/>
 
+  <suppress checks="com.azure.tools.checkstyle.checks.ThrowFromClientLoggerCheck"
+            files=".*AzureMonitorLogRecordExporter.java"/>
```
Comment (trask) [inline eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml:178]: can you add a comment above explaning this one?

**Review by trask — APPROVED** [review]

File: eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml (hunk 2 above)
Comment (jeanbisutti) [reply eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml]: It's a mistake. I shouldn't have pushed it.

**Pull request merged**